h2o-extensions/target-encoder/src/main/java/hex/api/targetencoding/TargetEncoderHandler.java

```diff
@@ -0,0 +1,49 @@
+package hex.api.targetencoding;
+
+import ai.h2o.targetencoding.BlendingParams;
+import ai.h2o.targetencoding.TargetEncoderModel;
+import hex.schemas.TargetEncoderTransformParametersV3;
+import water.Iced;
+import water.Key;
+import water.api.Handler;
+import water.api.schemas3.KeyV3;
+import water.fvec.Frame;
+
+public class TargetEncoderHandler extends Handler {
+  @SuppressWarnings("unused")
+  public KeyV3.FrameKeyV3 transform(final int version, final TargetEncoderTransformParametersV3 parametersV3) {
+    final TargetEncoderTransformParameters parameters = new TargetEncoderTransformParameters();
+    parametersV3.fillImpl(parameters);
+    final TargetEncoderModel model = parameters._model.get();
+
+    final boolean asTraining = parameters._as_training;
+    final boolean useBlending = parameters._blending;
+    final double noise = parameters._noise < -1  ? model._parms._noise : parameters._noise;
+    final double inflectionPoint = parameters._inflection_point < 0 ? model._parms._inflection_point : parameters._inflection_point;
```

Comment:
I wouldn't parse parameters that are not needed. I would hide this in `if useBlending` - makes the flow clearer.

Reply:
I see, you mean having a big `if (parameters._blending)` block instead? Sure, can do that.